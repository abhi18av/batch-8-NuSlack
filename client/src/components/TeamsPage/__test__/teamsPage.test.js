import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import TeamsPage from '../index';

describe('<TeamsPage /> shallow', () => {
  it('should match the snapshot', () => {
    const tree = shallow(<TeamsPage />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
