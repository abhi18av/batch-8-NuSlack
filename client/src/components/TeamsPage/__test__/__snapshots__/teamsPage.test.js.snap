// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TeamsPage /> shallow should match the snapshot 1`] = `
<Redirect
  push={true}
  to="/"
/>
`;
